src/core_plugins/kibana/public/management/sections/indices/_field_types.js

```diff
@@ -11,14 +11,21 @@ export default function GetFieldTypes() {
       scripted: 0
```

Comment:
Now that we're adding source filters, this feels like a poorly named file. We should probably just rename it as _edit_sections.js or something like that.